doc/deployment/migrations.md

```diff
@@ -1,45 +1,41 @@
-
-## Backup
-
-It’s paramount that you backup your data before running a migration.  While we’ve tested the migration code extensively, it’s still possible that they contain bugs, or that you accidentally use them in a wrong way.
-
-In general, there are two data storage systems that you might consider backing up: the metadata storage and the data storage. Not all migration scripts touch both systems, so you might only need to back up one of them. Look at the README for a particular migration script for details.
-
-### Backup the metadata store
-
-Assuming you’ve deployed Pachyderm on a public cloud, your metadata is probably stored on a persistent volume. See the respective [Deploying Pachyderm](http://pachyderm.readthedocs.io/en/stable/deployment/deploy_intro.html) guide for details.
-
-Here are official guides on backing up persistent volumes for each cloud provider:
-
-- [GCE Persistent Volume](https://cloud.google.com/compute/docs/disks/create-snapshots)
-- [Elastic Block Store (EBS)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-creating-snapshot.html)
-
-### Backup the object store 
-
-We don’t currently have migration scripts that affect the object store.
-
+New versions of Pachyderm often require a migration for some or all of the on
+disk objects. This document describes how Pachyderm migration works and
+the best practices surrounding it.
+
+As of 1.7, Pachyderm's migration works by extracting objects into a stream of
+API requests, and replaying those requests onto the newer version of pachd.
+This process happens automatically using Kubernetes' "rolling update"
+functionality. All you need to do is deploy Pachyderm (with `pachctl deploy) as
+you would if you were deploying for the first time. While migration is
+running you will see 2 pachd pods running, the one that was already
+running, and the new one. During this time the previous pod will still
+respond to requests, however write operations will race with the migration
+and may not make it to the new cluster. Thus you should make sure that all
+processes that write data to repos (i.e., call put-file) or create new
+pipelines are turned down before migration begins. You don't need to worry
+about pipelines running during the migration process.
+
+## Backups
+
+It is highly recommended that you backup your cluster before you perform
+a migration. This is accomplished with the `pachctl extract` command,
+which will generate a stream of API requests, the same stream that migrate
+uses, which will reconstruct your cluster. See the docs for [`pachctl
+extract`](http://docs.pachyderm.io/en/latest/pachctl/pachctl_extract.html)
+and [`pachctl
+restore`](http://docs.pachyderm.io/en/latest/pachctl/pachctl_restore.html)
+for further usage.
+
+
+## Before You Migrate.
+
+1.7 is the first Pachyderm version to support `extract` and `restore`
+which are necessary for migration. To solve this we've made a final 1.6
```

Comment:
"To solve this..." -> "To bridge the gap to previous Pachyderm versions, we've made a final..."